File: concrete/src/main/java/pcb/uwu/unit/quantity/KiloGramUnit.java
package pcb.uwu.unit.quantity;

import pcb.uwu.units.quantity.MassUnit;

import java.util.function.Function;

public class KiloGramUnit extends MassUnit {

	public static final KiloGramUnit KILOGRAM = new KiloGramUnit();

	private KiloGramUnit() {
		super("Kg", "kilogram", "kilograms",
				Function.identity(),
				Function.identity());
	}
}

File: implementation/src/main/java/pcb/uwu/unit/base/KiloGramUnit.kt
package pcb.uwu.unit.base

import pcb.uwu.util.identity

object KiloGramUnit
    : MassUnit(symbol = "Kg",
               singularName = "kilogram",
               pluralName = "kilograms",
               translationToCanonical = identity(),
               translationFromCanonical = identity())
